crates/tests-e2e/tests/force_close_position.rs

```diff
@@ -0,0 +1,51 @@
+use tests_e2e::bitcoind::Bitcoind;
+use tests_e2e::coordinator::Coordinator;
+use tests_e2e::coordinator::SubChannelState;
+use tests_e2e::setup;
+use tests_e2e::wait_until;
+
+#[tokio::test]
+#[ignore = "need to be run with 'just e2e' command"]
+async fn can_force_close_position() {
+    let test = setup::TestSetup::new_with_open_position().await;
+    let coordinator = &test.coordinator;
+    let app = &test.app;
+    let bitcoin = &test.bitcoind;
+
+    wait_until!(app.rx.node_pubkey().is_some());
+    let app_pubkey = app.rx.node_pubkey().unwrap();
```

Comment:
(see other comment for explanation).

```suggestion
    let app_pubkey = api::get_node_id().0;
```

Reply:
Nice, I didn't know this. Thanks.